src/Main.java

```diff
@@ -0,0 +1,24 @@
+package src;
+
+import java.util.ArrayList;
+
+public class Main {
+    public static void main(String[] args) {
+        doubleTypePokemon b2 = new doubleTypePokemon("Bulbasaur", Type.Grass, 5000, 5,10, Type.Poison, 120, 20, new Stat(110,5,8));
```

Comment:
CLass namen met hoofdletter is zodat je hier duidelijk ziet dat DoubleTypePokemon de class is en b2 het object. Het helpt je zelf en je collega's!